server/app-engine/static/vehicle-assignments.js

```diff
@@ -87,24 +87,6 @@ function showToast(text) {
     }, 2000);
 }
 
-function handleModal(name) {
```

Comment:
It looks like you made actual code changes to the dispatch UI. Please be sure you've done enough testing for these changes.